### PR TITLE
Update XM-Personalize FAQ content

### PR DESCRIPTION
**Pull request by jst-cyr**
## Description / Motivation
Addresses #755 . Largely used to update the FAQ to use proper headings (for direct linking) and to remove 'future state' content that is either now released or no longer relevant. Some other irrelevant content also removed as part of cleanup.

## How Has This Been Tested?
Local and [on Vercel](https://developer-portal-cp9tmaeq2-sitecoretechnicalmarketing.vercel.app/learn/faq/xm-personalize)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation update (non-breaking change; modified files are limited to the `/data` directory or other markdown files)

## Checklist:
- [X] I have read the Contributing guide.
- [X] My code/comments/docs fully adhere to the Code of Conduct.
- [] My change is a code change.
- [X] My change is a documentation change and there are NO other updates required.
- [ ] My change has new or updated images which are stored in the `/public/images` folder that need to be migrated to Sitecore DAM

### CODE REVIEW

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/features.md [hunk 1]
```diff
@@ -6,45 +6,35 @@ hasSubPageNav: true
...
 
 There is a preview tool in Personalize to test and view draft and live experiences. Customers can also see the data there in terms of performance, impact, etc.
 
-**Are we able to modify the order form for the XM2XMP bundled offering?**
-
-The Order Form must remain &quot;as-is&quot; to include all product tables. The Order Form will still have itemized tables for each individual product vs. having one table listing all products within the bundle. On the flip side, there is no requirement to have multiple Order Forms, only one is required, with individual product tables. Due to the different SLAs currently in place, products need to be broken out to protect Sitecore from a contractual and legal perspective.
-
-**Can we stay on our current hosting solution (not MC) until XM Cloud is released?**
+### Can we stay on our current hosting solution (not MC) until XM Cloud is released?
```
[the remark this answers is not in the document]
Comment (dylanyoung-dev) [reply apps/devportal/data/markdown/pages/learn/faq/xm-personalize/features.md:29]: Also not sure about this question, because it says "until XM Cloud is released" but it already has been released.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/features.md [hunk 2]
```diff
@@ -6,45 +6,35 @@ hasSubPageNav: true
 cdpTags: ['xm', 'personalize']
 ---
 
-**Once I build a XM + Personalize site and deploy it, will I be able to port this site to XM Cloud when that comes out? Do you have a playbook available that we could see?**
+### Once I build a XM + Personalize site and deploy it, will I be able to port this site to XM Cloud when that comes out? Do you have a playbook available that we could see?
```
Comment (dylanyoung-dev) [inline apps/devportal/data/markdown/pages/learn/faq/xm-personalize/features.md:9]: This sentence mentions `to XM Cloud when that comes out?` Probably should just rephrase.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md [hunk 3]
```diff
@@ -6,73 +6,65 @@ hasSubPageNav: true
...
 
-**Does Personalize work at the component level to swap out content or just pop-ups? **
+### Does Personalize work at the component level to swap out content or just pop-ups?
 
 Personalization is not restricted to just pop-up functionality but instead can apply to page level or component level variations. Offers stored in Sitecore Personalize paired with Web Experiences can allow marketers to customize various parts of a webpage, from within Sitecore Personalize using Html/CSS/JavaScript. In addition, using a Full Stack headless experience, any content irrespective of the source (including XM or Content Hub) can be personalized.
 
-**Will my potential Personalize implementation still be compatible and work if I later move into XM SaaS?**
+### Will my potential Personalize implementation still be compatible and work if I later move into XM Cloud?
 
 If the user has purchased Standalone Personalize (or CDP, SmartHub CDP) then they will keep that instance if they move to XM Cloud. The embedded Personalization in Pages will be enhanced with additional condition templates and the user will be able to update the experiences created in Pages by going into the Sitecore Personalize app.
```
Comment (dylanyoung-dev) [inline apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md:56]: Weird, I don't really see this in the code, but there is a ton of odd white space underneath a few of the questions on this page.  So I'm not sure if there is much that could be done, but something to look into.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md [hunk 4]
```diff
@@ -6,73 +6,65 @@ hasSubPageNav: true
...
-**Future - Coming Soon, not currently Available**
-Experience Targeting: We want the marketer persona to be able to set up an experience that delivers different content based on a pre-defined audience. This pre-defined audience is built upon the same structure as XM/P conditions. The user can combine these conditions into an audience and decide on what output the variant will return. We have high level plans to build some level of integration between XM and Personalize so that the user could set up personalization in XM and it automatically creates an experience with the required variants and assigned audiences. There are some rules in XM that won&#39;t be transferrable. More work to be done here.
-
-**We currently use XM Managed Cloud but like the sound of adding Personalize and Send. We use .Net to publish to the web apps in Azure, will we continue to do that if we move to your new XP2XMP offering?**
+### We currently use XM Managed Cloud but like the sound of adding Personalize and Send. We use .Net to publish to the web apps in Azure, will we continue to do that if we move to your new XP2XMP offering?
```
Comment (dylanyoung-dev) [inline apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md:33]: There are references to Send in weird places, I'd suggest but not required, removing all references to Send, since this is purely about XM + Personalize.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md [hunk 5]
```diff
@@ -92,10 +84,9 @@ Regarding a full description, please refer to the documentation website. The fol
 **Real-Life Example (Jetstar Airways &amp; Sitecore)**
 
 - [https://www.sitecore.com/blog/personalization/the-skys-the-limit-for-jetstar-airways ](https://www.sitecore.com/blog/personalization/the-skys-the-limit-for-jetstar-airways%C2%A0)
-- [https://sitecore.showpad.biz/webapp2/results?query=vercel%20pitch&amp;scope=content,coaching&amp;slug=0857aedc7594214236e1967f7837fd27 ](https://sitecore.showpad.biz/webapp2/results?query=vercel%20pitch&scope=content,coaching&slug=0857aedc7594214236e1967f7837fd27%C2%A0)
 
 To solve a challenge of Create and act on insights from a single customer profile driven by an omnichannel experience strategy.**Jetstar** Use Sitecore Experience Manager as the core digital transformation platform, Sitecore CDP to activate customer insights and Sitecore Personalize deployed across multiple channels.
```
Comment (dylanyoung-dev) [inline apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md:88]: There is a missing space here between the period "." and Jetstar.  Also `Use` is capitalized when I don't think it should be (following the word Jetstar.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md [hunk 6]
```diff
@@ -92,10 +84,9 @@ Regarding a full description, please refer to the documentation website. The fol
 **Real-Life Example (Jetstar Airways &amp; Sitecore)**
```
Comment (dylanyoung-dev) [inline apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md:84]: Above in #1 and #2 references to Sitecore CDP should really be replaced with Sitecore Personalize.  Especially since this is an XM + P document, but also could give users the belief that CDP is needed for this, but all of this is Personalize.  Lots of documents I've found use CDP/P interchangeably, and sometimes you probably could, but this time you probably shouldn't because it creates confusion.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/features.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jst-cyr) [reply apps/devportal/data/markdown/pages/learn/faq/xm-personalize/features.md:29]: Good point! I'm going to spell MC out in full and remove the "until XM Cloud is released" as it is unnecessary.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/features.md (hunk 2 above)
Comment (jst-cyr) [reply apps/devportal/data/markdown/pages/learn/faq/xm-personalize/features.md:9]: My next commit will include a removal of that part. The sentence works without it

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md (hunk 3 above)
Comment (jst-cyr) [reply apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md:56]: @dylanyoung-dev : Can you provide more details on this? I can see some headings that are the wrong level (h2 vs h3) but I'm not sure if I'm addressing the right issues on this page.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md (hunk 4 above)
Comment (jst-cyr) [reply apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md:33]: Good point, removing the references to Send in all the pages (except for Features). My guess is that the original packaging of the offer was to give Personalize and Send to replace XP, but that's not the context here anymore.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md [hunk 7]
```diff
@@ -6,73 +6,65 @@ hasSubPageNav: true
...
 
 With an existing XP or XM on-prem implementation, to start utilizing Sitecore Personalize, you will need to integrate the web tracking mechanisms into your existing XP/XM instance. This includes placing the Sitecore CDP/Personalize JS Library (Boxever.js) and then adding tracking events in JavaScript, to track view and identity events where appropriate. There are details about these required changes in the Developer Portal ([https://developers.sitecore.com/learn/integrations/xm-smarthub-cdp](https://developers.sitecore.com/learn/integrations/xm-smarthub-cdp))
 
-**I see that Sitecore Personalize (SP) has content (offers), can you store these in Content Hub or XM? Is there a single source of content?**
+## I see that Sitecore Personalize (SP) has content (offers), can you store these in Content Hub or XM? Is there a single source of content?
 
 No, you currently cannot store offers in Content Hub or XM and pull these into Sitecore Personalize. However, you can pull content from XM or Content Hub when personalizing a piece of content using a Full Stack experience vs a Web Experience using custom code in your front end.
```
Comment (dylanyoung-dev) [inline apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md:65]: I would maybe rephrase this whole section.. Here's what I have using Chat GPT:

There are two approaches to handling a single content source. The first approach involves using a Decision Model with a Web Experience, which directly pulls content from your content management system within the Decision Model, and then displays the result in your Web Experience. The second approach is to use a Full Stack Interactive experience, where your frontend code requests a decision, receives a JSON result, and then retrieves the correct content items.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/scaling.md [hunk 8]
```diff
@@ -21,12 +21,12 @@ The XP2XMP product bundle allows customers to deploy Sitecore Experience Manager
...
 
 We believe Jamstack implementations are the future of digital experiences and we anticipate enterprises will transition to these implementations as part of their site refresh cadence. One of the benefits of Jamstack client applications is the ability to render large portions of the sites as static pages which can be hosted on CDNs or dedicated application edge providers such as Vercel, Netlify, and Cloudflare. These edge solutions make Time-to-First-Byte (TTFB), First Contentful Paint (FCP), and Time to Interactive (TTI) for static pages as short as less than 1 second, making a huge difference for the visitor experience and significantly increasing the likelihood of achieving conversion goals.
 
-**What is testing and personalization &#39;across the funnel&#39;?**
+### What is testing and personalization &#39;across the funnel&#39;?
 
 This refers to Personalization &amp;Testing experiences that can run across multiple pages and on experiences that are not being powered by Sitecore today. This is a powerful feature not available in XP that provides more extensibility of the Sitecore Personalize solution and can help organizations demonstrate ROI quicker.
```
Comment (dylanyoung-dev) [inline apps/devportal/data/markdown/pages/learn/faq/xm-personalize/scaling.md:32]: Needs to be a space after &amp; and before Testing.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/edge-vercel.md [hunk 9]
```diff
@@ -6,16 +6,17 @@ hasSubPageNav: true
...
 
 This will not necessarily secure the application by default. When moving to SaaS there are some components of the Sitecore XM Solution that the customer will have to give up control over to move to SaaS. So, no.
 
-Some examples of XM capabilities that will not be available in XM Cloud are custom Solr indexes and use of Sitecore Forms. The list of XM capabilities which will not be available in XM Cloud will follow in the upcoming month.
+Some examples of XM capabilities that will not be available in XM Cloud are custom Solr indexes and use of Sitecore Forms.
+
+### I already pay for Vercel for other applications we have, can I use my Vercel with XP2XMP?
 
-**I already pay for Vercel for other applications we have, can I use my Vercel with XP2XMP?**
 You can use your own Vercel subscription with XP2XMP is you are implementing a headless CMS.
```
Comment (dylanyoung-dev) [inline apps/devportal/data/markdown/pages/learn/faq/xm-personalize/edge-vercel.md:17]: Replace `is` with `if`

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/development.md [hunk 10]
```diff
@@ -6,36 +6,36 @@ hasInPageNav: false
...
 
 When using Sitecore Experience Manager from the XP2XMP bundle with the Sitecore JavaScript Rendering SDK (JSS), developers can use Server-Side Rendering (SSR) with their Next.js, React, Angular, or Vue.js applications. React, Angular, and Vue.js require customers to host an [SSR proxy](https://doc.sitecore.com/xp/en/developers/hd/190/sitecore-headless-development/server-side-render-jss-apps-headlessly-using-the-jss-proxy.html) server themselves to support SSR with JSS. Next.js supports SSR without requiring a separate SSR proxy server as this capability is supported through Next.js directly. This is one of the reasons why Sitecore recommends using Next.js as the Jamstack framework for client applications when implementing a headless solution.
 
-**We just launched our XM site on 10.1 using the best practices of SXA. Can we port that site over to XP2XMP so we can take advantage of Personalize and Send that we did not purchase with our XM License?**
+### We just launched our XM site on 10.1 using the best practices of SXA. Can we port that site over to XP2XMP so we can take advantage of Personalize and Send that we did not purchase with our XM License?
```
Comment (dylanyoung-dev) [inline apps/devportal/data/markdown/pages/learn/faq/xm-personalize/development.md:19]: Mentions of Send in a XM + Personalize document, probably doesn't make sense?

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/customization.md [hunk 11]
```diff
@@ -6,14 +6,14 @@ hasSubPageNav: true
...
 
 MVC websites that are self-hosted or run in Sitecore Managed Cloud with the XP2XMP product bundle should continue to support existing .NET MVC components.
 
-**We use Sitecore Search and have customized the indexes. Will this work in XP2XMP and XM Cloud?**
+### We use Sitecore XM's content search API and have customized the indexes. Will this work in XP2XMP and XM Cloud?
 
 Customers will be able to create custom indexes if they host Sitecore Experience Manager in their own Cloud or use Sitecore Managed Cloud. With XM Cloud, customers will not be able to create custom indexes as Solr will be fully controlled by the XM Cloud platform. Sitecore recommends that customers use a third-party search provider if custom indexes are required.
 
-**What extension possibilities/flexibility for custom business requirements are there for XM, Send, Personalize?**
+### What extension possibilities/flexibility for custom business requirements are there for XM, Send, Personalize?
```
Comment (dylanyoung-dev) [inline apps/devportal/data/markdown/pages/learn/faq/xm-personalize/customization.md:17]: Another mention to Send, not sure it makes sense in an XM + Personalize document, since it has it's own considerations.

**Review by dylanyoung-dev — CHANGES_REQUESTED** [review]
Alright I've left a bunch of things, up to your discretion on some of them, while others are more important.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md (hunk 5 above)
Comment (jst-cyr) [reply apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md:88]: The whole paragraph reads awkwardly, but I've made the fixes you highlighted as part of the next push.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md (hunk 6 above)
Comment (jst-cyr) [reply apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md:84]: Got it, will replace CDP with Personalize in the first section. I did not see references to CDP in the second use case.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md (hunk 7 above)
Comment (jst-cyr) [reply apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md:65]: I have updated the answer to **Is there a single source of content?** with the text you supplied

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/edge-vercel.md (hunk 9 above)
Comment (jst-cyr) [reply apps/devportal/data/markdown/pages/learn/faq/xm-personalize/edge-vercel.md:17]: Fixed in the next batch!

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/edge-vercel.md
[the remark this answers is not in the document]
Comment (jst-cyr) [reply apps/devportal/data/markdown/pages/learn/faq/xm-personalize/edge-vercel.md:1]: This is going to be a big rewrite... there are a lot of VERY specific Vercel pitch stuff in here. I will take a stab at a rewrite but will likely need some help from @thomas-desmond to review.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/development.md (hunk 10 above)
Comment (jst-cyr) [reply apps/devportal/data/markdown/pages/learn/faq/xm-personalize/development.md:19]: Addressed while working on the other Send references.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md (hunk 3 above)
Comment (jst-cyr) [reply apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md:56]: Marking this as resolved as the headings change seems to make it look as expected.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md (hunk 5 above)
Comment (dylanyoung-dev) [reply apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md]: I see what you mean.  Really that jetstart section should read (with the help of ChatGPT)

To address the complexity of deriving and implementing insights from a unified customer profile in an omnichannel experience strategy, Jetstar leverages Sitecore Experience Manager as the main platform for digital transformation. It also uses Sitecore CDP to activate customer insights and Sitecore Personalize across multiple channels.

**Review by dylanyoung-dev — COMMENTED** [review]
Everything looks good, but I did provide a suggestion on improving that sentence related to Jetstar.  Your welcome to fix that with my suggestion or skip.  Up to you of course.

File: apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md (hunk 5 above)
Comment (jst-cyr) [reply apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md]: I took this but changed 'leverages' to 'uses'. Otherwise, let's call this done!

**Comment by jst-cyr** [comment]
Force merging since Dylan has already signed off, but I did include the suggested change as well.

**Comment by dylanyoung-dev** [comment]
lol beat me to it, but I just looked and those changes look good!
